(chore #54) Improve environment variables documentation (#55)
medic/cht-sync: README.md

- `Docker`
- An `.env` file containing the environment variable placeholders from the `.env.template` file. The file should be located in the root directory of the project or set by the operating system. The variables should be customized accordingly for the specific deployment needs.

#### Environment variable

> We have four environment variable groups in the `.env.template` file. It is important to understand the difference between them.

Reviewer: ```suggestion
There are four environment variable groups in the `.env.template` file. To successfully set up `cht-sync`, It is important to understand the difference between them.
```

Reviewer: Added a clarification on why it is important to know the difference between the variables, and removed the "we" mention.

> 1. Postgresql and Postgres: Are used to establish the Postgres database to synchronize CouchDB data. It also define the schema and table names to store the CouchDB data. The main objective is to define the environment where the raw CouchDB data is going to be copy to.

Reviewer: ```suggestion
1. Postgresql and Postgres: Are used to establish the Postgres database to synchronize CouchDB data. They also define the schema and table names to store the CouchDB data. The main objective is to define the environment where the raw CouchDB data will be copied.
```

Reviewer: @lorerod I don't see a `Postgresql` variable in the `.env.template`. Where do we use that variable? Am I missing something?

Author: Both applications use the same set of environment variables.

Reviewer: Aaah makes sense, thanks for the explanation!

Reviewer: There is this last suggestion to review/solve before I can approve. Can you please look into it, @lorerod ? It's a typo suggestion.

Author: Yes! Thank you so much, Andra, for looking into this.

> 2. DBT: These environment variables are exclusive to the DBT configuration. The main objective is to define the environment where the tables and views for the models defined in `CHT_PIPELINE_BRANCH_URL` will be created. It is important to separate this environments from the previous group. `DBT_POSTGRES_USER` and `DBT_POSTGRES_SCHEMA` must be different from `POSTGRES_USER` and `POSTGRES_SCHEMA`. `DBT_POSTGRES_HOST` has to be the Postgres instance created with the environment variables set in the first group.

Reviewer: ```suggestion
2. DBT: These environment variables are exclusive to the DBT configuration. The main objective is to define the environment where the tables and views for the models defined in `CHT_PIPELINE_BRANCH_URL` will be created. It is important to separate this environment from the previous group. `DBT_POSTGRES_USER` and `DBT_POSTGRES_SCHEMA` must be different from `POSTGRES_USER` and `POSTGRES_SCHEMA`. `DBT_POSTGRES_HOST` has to be the Postgres instance created with the environment variables set in the first group.
```
